elixir/apps/web/lib/web/live/actors/groups.ex

```diff
@@ -0,0 +1,247 @@
+defmodule Web.Actors.Groups do
```

Comment:
```suggestion
defmodule Web.Actors.EditActorGroups do
```

Reply:
As mentioned in another comment, I went with `EditGroups` for now.  If you'd rather it be `EditActorGroups`, just let me know.